source/SIL.AppBuilder.Portal.Frontend/src/lib/string/unit-test.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { tokensToObject } from './utils';
+
+describe('Unit | string', () => {
```

Comment:
@giancorzo if you need to do unit tests, you could follow this pattern.

Reply:
+1